partcad/src/partcad/actions/part_actions.py

```
@@ -0,0 +1,77 @@
+import shutil
+from pathlib import Path
+import partcad.logging as logging
+
+def add_part_action(project, kind, path, config=None):
```

Ruff (0.8.2)

Missing return type annotation for public function `add_part_action`

Add return type annotation: `None`

(ANN201)